### PR TITLE
Add Control-g keybinding to list *all* files

### PR DESCRIPTION
**Pull request by nathanshelly**
My take on #1

### CODE REVIEW

File: functions/_fzf_functions [hunk 1]
```diff
@@ -1,7 +1,15 @@
 #! /usr/bin/env sh
 
+# default fzf functions - `$(brew --prefix)/opt/fzf/shell/key-bindings.zsh`
+
 # <<<< widgets >>>>
 
+# list ignored files
+# basically just `fzf-file-widget` with `--no-ignore`
+fzf-ignored-file-widget() {
+  FZF_CTRL_T_COMMAND="${FZF_DEFAULT_COMMAND} --no-ignore" fzf-file-widget
```
Comment (glenshelly) [inline functions/_fzf_functions]: Interesting.  I confess to not understanding why this line of code even works - I'd expect... a semi-colon before fzf-file-widget.  And, wouldn't this change the value of FZF_CTRL_T_COMMAND for future uses, or are we in some kind of "subscope" that only lives for the life of the single execution?

File: functions/_fzf_functions [hunk 2]
```diff
@@ -1,7 +1,15 @@
 #! /usr/bin/env sh
```
Comment (glenshelly) [inline functions/_fzf_functions]: are some of these functions defined here used in multiple places?  As a contributor, I found it a bit confusing to need to define my fzf functions in one directory, and use them in another, hence my focus on the single fzf.zsh class.  Beyond grouping any and all functions into the /functions directory, is there another reason for putting the fzf functions in a directory other than /utilities/fzf?

File: functions/_fzf_functions (hunk 1 above)
Comment (glenshelly) [reply functions/_fzf_functions]: Ok, reading up, so, sure, we're just setting the FZF_CTRL_T_COMMAND locally, which doesn't affect the global environment.  But... no need for a semi-colon?

Comment (nathanshelly) [reply functions/_fzf_functions]: yeah these assignments are in scope only for the specific command. This [StackOverflow post](https://unix.stackexchange.com/questions/126938/why-is-setting-a-variable-before-a-command-legal-in-bash) does a good job explaining what's going on.

File: functions/_fzf_functions (hunk 2 above)
Comment (nathanshelly) [reply functions/_fzf_functions]: TLDR: That is a great critique. Current organization began with different concerns and I haven’t reevaluated my organization as my dotfiles have evolved. I don’t have immediate plans to reorganize but I’ve made a note to change to something more like your suggestion in the future.

Long version: my dotfiles organizational structure comes mostly from [Holman’s dotfiles](https://github.com/holman/dotfiles). His structure has a `functions` folder of which all scripts are automatically added to the path (see [this line](https://github.com/nathanshelly/.files/blob/master/zsh/zshrc.symlink#L55)). As I’ve added more functions I never really reconsidered where to put them so I defaulted to the functions folder. When I went to make this PR I already had `fzf` related functions in the `$DOTFILES/functions/_fzf_functions` file so I defaulted to putting the new `fzf-ignored-files-widget` in that file.

**Pull request merged**